src/fsharp/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.Utilities.fs

```diff
@@ -152,16 +152,16 @@ module internal Utilities =
         else
             None
 
-    let drainStreamToFile (stream: StreamReader) filename =
-        use file = File.OpenWrite(filename)
-        use writer = new StreamWriter(file)
+    let drainStreamToMemory (stream: StreamReader) =
+        let mutable list = List.empty
         let rec copyLines () =
             match stream.ReadLine() with
             | null -> ()
             | line ->
-                writer.WriteLine(line)
+                list <- line :: list
                 copyLines ()
         copyLines ()
+        list |> List.toArray
```

Comment:
I think a `ResizeArray` would be preferred here since you seem to be effectively copying the semantics of one but with the F# list type